src/main/java/tasks/homework/streamtask/MyStreamClass.java

```diff
@@ -13,27 +13,29 @@
 
     public static void main(String[] args) {
 
-        List<Integer> numbers = Arrays.asList(1, 2, 3, 4 , 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21);
-        List<Integer> numberStream = numbers.stream()
+        List<Integer> numbers = Stream.iterate(1, n -> n + 1)
+                .limit(20)
                 .filter(number -> number % 2 == 0)
```

Comment:
why add all numbers when to delete half of them. You can add straight away only odd ones